docker/build_image.sh

```diff
@@ -64,6 +66,17 @@ do
           BUILD_WITH_IPEX=true
           shift
           ;;
+        -py|--pythonversion)
+          PYTHON_VERSION="$2"
+          if [[ $PYTHON_VERSION = 3.8 || $PYTHON_VERSION = 3.9 || $PYTHON_VERSION = 3.10 ]]; then
+            echo "Valid python version"
+          else
+            echo "Valid python versions are 3.8, 3.9 and 3.10"
```

Comment:
I think we can go ahead and support 3.11 as well, If someone is trying to torch.compile something with 3.11 they will get an error thrown from torch directly

Reply:
I will add docker support for Python 3.11 as a broader PR for TorchServe python 3.11 support